src/main/java/de/dhbw/mh/slang/codegen/JavaBytecodeGenerator.java

```diff
@@ -97,7 +115,16 @@ public String visitPost( AstBinaryOperation node, String lhs, String rhs ){
 				throw new RuntimeException( "not yet implemented" );
 			}
 			case LESS_OR_EQUAL:{
-				throw new RuntimeException( "not yet implemented" );
+				StringBuilder builder = new StringBuilder();
+				builder.append(lhs);
+				builder.append(rhs);
+				builder.append(String.format("if_icmpgt #0_gt%n"));
+				builder.append(String.format("iconst_1%n"));
+				builder.append(String.format("goto #0_end%n"));
+				builder.append(String.format("#0_gt:%n"));
+				builder.append(String.format("iconst_0%n"));
+				builder.append(String.format("#0_end:%n"));
```

Comment:
A counter would be helpful to increment the label ID. However, I did not code this requirement into the test and I am also happy not to resolve too many merge conflicts 😄